public/themes/mocha.css

```
@@ -0,0 +1,1627 @@
+/* Spectate button styling */
+.spectate-button {
+  background-color: var(--spectate-button-background-color) !important;
+  color: var(--spectate-button-color) !important;
+  border: 1px solid var(--primary)
+    /* Orange border */
+    !important;
+  font-weight: bold;
+  border-radius: 5px;
+  transition: all 0.3s ease;
+  font-size: 14px;
+  transition: background-color 0.3s ease, color 0.3s ease;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

transition is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)